src/Microsoft.AspNetCore.Build/NuGetInstall.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using NuGet.Packaging;
+using NuGet.Packaging.Core;
+
+namespace Microsoft.AspNetCore.Build
+{
+    public class NuGetInstall : Task
+    {
+        [Required]
+        public ITaskItem[] Packages { get; set; }
+
+        [Required]
+        public string DestinationFolder { get; set; }
+
+        public override bool Execute()
+        {
+            if (!Directory.Exists(DestinationFolder))
+            {
+                Directory.CreateDirectory(DestinationFolder);
+                Log.LogMessage($"Created destination folder: {DestinationFolder}");
+            }
+
+            var success = true;
+            var logger = new MSBuildNuGetLogger(Log);
+            foreach (var item in Packages)
+            {
+                // Load the package
+                var path = item.GetMetadata("FullPath");
+                if (!File.Exists(path))
+                {
+                    Log.LogError($"Package file not found: {path}");
+                    success = false;
```

Comment:
Do you still care about performing remaining work if you hit this condition? Can't you just break here?